src/lib/y2network/widgets/slave_items.rb

```diff
@@ -29,6 +27,10 @@ module SlaveItems
       include Yast::Logger
       include Yast::I18n
 
+      def initialize
+        Yast.import "NetworkInterfaces"
```

Comment:
@jreidinger maybe you prefer to omit this change, if not, then we could move the `textomain "network"` here like we usually do.

Reply:
fine for now. For future I am not sure if we really need it.

Reply:
I just removed the LanItems import and left it as it was..